hugo/content/exporter/_index.md

```diff
@@ -520,3 +479,151 @@ Collecting all per-query metrics into Prometheus could greatly increase storage
 ### System {#system}
 
 \*NIX Operating System metrics (Linux, BSD, etc) are collected using the [node_exporter](https://github.com/prometheus/node_exporter) provided by the Prometheus team. pgMonitor only collects the default metrics provided by node_exporter, but many additional metrics are available if needed.
+
+
+## Legacy postgres_exporter Setup {#postgres-exporter}
+
+If you had been using pgMonitor prior to version 5.0.0, postgres_exporter was the method used to collect PostgreSQL metrics. This exporter can still be used with 5.0.0, but there are some additional steps required. It is HIGHLY recommended to switch to using sql_exporter as soon as possible. Custom query support will be dropped from postgres_exporter and no new features of pgMonitor are being developed around it.
```

Comment:
You mention that custom query support is being dropped, but you don't explain that that would completely break pgM since it relies on it. So I'd suggest either removing that comment in the multiple places you have it, or expand it to explain why it matters. (If it were me, I'd just drop it everywhere)

Reply:
I updated this to explain why